### ISSUE TITLE
PageView Route

### ISSUE THREAD

**Issue by mmaelicke** [sumo-rhine/kinamo#13]
Implement a `<PageViewRoute />` to automatically send `page_view` events to ga. Implement after #6 is finished and the backend is actually receiving data.

**Comment by mmaelicke** [comment]
done with `0.0.19`, ut I already think that it is a bad idea to do it this way.
@JesJehle, we might need to re-open this or a new issue, once version 0.1 is released.
